apis/Google.Cloud.Logging.V2/Google.Cloud.Logging.Log4Net/MemoryLogQueue.cs

```diff
@@ -68,42 +69,23 @@ public DateTimeRange EnqueueAsync(IEnumerable<LogEntryExtra> logEntries)
                     }
                     var sizedEntry = new LogEntryExtraSize(entry, _maxMemorySize > 0 ? entry.Entry.CalculateSize() : 0);
                     _q.Enqueue(sizedEntry);
-                    if (entry.Id > _maxIdSeen)
```

Comment:
Why are we removing this block? (I'm not sure I've got a good "sense" of this PR...)